src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -60,7 +61,15 @@ public StoreController(@NotNull final StoreService storeService) {
   @Override
   public ResponseEntity<Void> storeProduct(
       final String acceptVersion, @Valid final MultipartFile file) {
-    // TODO validate Accept-Version
+    final String expectedAcceptVersion = storeApiVersion;
+    if (!StringUtils.equals(acceptVersion, expectedAcceptVersion)) {
+      log.warn(
+          "Excepted Accept-Version to be \"{}\" but was \"{}\". Only \"{}\" is currently supported.",
+          expectedAcceptVersion,
+          acceptVersion,
+          expectedAcceptVersion);
```

Comment:
❓ Would HTTP 412 Precondition Failed be a better status code here?

Reply:
We should support backwards compatibility, but we currently don't do that. Because of this, I don't think that 412 is appropriate, but maybe there is something better than 501.

Reply:
In these early days I don't think it matters too much as long as it isn't in the 200s.